macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -480,3 +531,26 @@ fileprivate extension CGPath {
 		return CGPath(ellipseIn: NSRect(x: 0, y: 0, width: diameter, height: diameter), transform: nil)
 	}
 }
+
+fileprivate extension NSColor {
```

Comment:
Isn't swiftlint complaining about this fileprivate access modifier?

Reply:
Not sure why isn't this working. Seems like it's enabled [here](https://github.com/microsoft/fluentui-apple/blob/93264f87f66d89075915da14618d934924da6d60/.swiftlint.yml#L84). 

I also see extensive use of file private modifier across the [code](https://github.com/microsoft/fluentui-apple/search?q=fileprivate)